install/empty_data.php

```diff
@@ -5319,7 +5359,56 @@ public function getEmptyData(): array
                     ##lang.ticket.reminder.bumptotal##: ##ticket.reminder.bumptotal##&lt;/a&gt;&lt;br /&gt;
                     ##lang.ticket.reminder.deadline##: ##ticket.reminder.deadline##&lt;/p&gt;
                     &lt;p&gt;##lang.ticket.reminder.text##: ##ticket.reminder.text##&lt;/p&gt;',
-            ]
+            ], [
+                'id' => '32',
+                'notificationtemplates_id' => '32',
+                'language' => '',
+                'subject' => '##knowbaseitem.action## - ##knowbaseitem.subject##',
+                'content_text' => '##lang.knowbaseitem.url## : ##knowbaseitem.url##
+
+                ##lang.knowbaseitem.subject## : ##knowbaseitem.subject##
+
+                ##lang.knowbaseitem.content## : ##knowbaseitem.content##
+
+                ##lang.knowbaseitem.categories## : ##knowbaseitem.categories##
+                ##lang.knowbaseitem.is_faq## ##knowbaseitem.is_faq##
+                ##lang.knowbaseitem.begin_date## : ##knowbaseitem.begin_date##
+                ##lang.knowbaseitem.end_date## : ##knowbaseitem.end_date##
+
+                ##lang.knowbaseitem.numberofdocuments## : ##knowbaseitem.numberofdocuments##
+
+                ##FOREACHdocuments##
+                    ##lang.document.downloadurl## : ##document.downloadurl##
+                    ##lang.document.filename## : ##document.filename##
+                    ##lang.document.heading## : ##document.heading##
+                    ##lang.document.id## : ##document.id##
+                    ##lang.document.name## : ##document.name##
+                    ##lang.document.url## : ##document.url##
+                    ##lang.document.weblink## : ##document.weblink##
+                ##ENDFOREACHdocuments##
+
+                ##FOREACHtargets##
+                    ##lang.target.itemtype## : ##target.type##
+                    ##lang.target.name## : ##target.name##
+                    ##lang.target.url## : ##target.url##
+                ##ENDFOREACHtargets##',
```

Comment:
Plain text should not be indented.

```suggestion

##lang.knowbaseitem.subject## : ##knowbaseitem.subject##

##lang.knowbaseitem.content## : ##knowbaseitem.content##

##lang.knowbaseitem.categories## : ##knowbaseitem.categories##
##lang.knowbaseitem.is_faq## ##knowbaseitem.is_faq##
##lang.knowbaseitem.begin_date## : ##knowbaseitem.begin_date##
##lang.knowbaseitem.end_date## : ##knowbaseitem.end_date##

##lang.knowbaseitem.numberofdocuments## : ##knowbaseitem.numberofdocuments##

##FOREACHdocuments##
  ##lang.document.downloadurl## : ##document.downloadurl##
  ##lang.document.filename## : ##document.filename##
  ##lang.document.heading## : ##document.heading##
  ##lang.document.id## : ##document.id##
  ##lang.document.name## : ##document.name##
  ##lang.document.url## : ##document.url##
  ##lang.document.weblink## : ##document.weblink##
##ENDFOREACHdocuments##

##FOREACHtargets##
  ##lang.target.itemtype## : ##target.type##
  ##lang.target.name## : ##target.name##
  ##lang.target.url## : ##target.url##
##ENDFOREACHtargets##',
```